### ISSUE TITLE
Remove the vpool if the volumedriver is not running

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#850]
We should be able to remove the vpool if the vpool is not running. We experienced this error during a test:

```
2016-08-24 17:29:03 43500 +0200 - stor-04.be-g8-3 - 4854/139859400984384 - celery/celery.worker.job - 930 - ERROR - Task ovs.storagerouter.remove_storagedriver[7c3022c9-0447-4a78-aa1e-029b1b6dcaab] raised unexpected: RuntimeError('Not all StorageDrivers are reachable, please (re)start them and try again',)
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "/opt/OpenvStorage/ovs/lib/storagerouter.py", line 1102, in remove_storagedriver
    raise RuntimeError('Not all StorageDrivers are reachable, please (re)start them and try again')
RuntimeError: Not all StorageDrivers are reachable, please (re)start them and try again
```

**Comment by khenderick** [comment]
An important remark: This means that the code will have to assume that a non-responsive StorageDriver is in fact down (stopped state) as it won't be notified of changes in the cluster and we assume that since it's down, it will reload configuration settings as soon as it's started and will join the cluster again.

**Comment by khenderick** [comment]
Fixed by #1232, packaged in openvstorage-2.7.7-rev.4391.7d827d6

**Comment by kinvaris** [comment]
### Status
PASSED

### OS:
`Ubuntu 16.04`

### Packages:
```
ii  blktap-openvstorage-utils            2.0.90-2ubuntu4                     amd64        utilities to work with VHD disk images files
ii  libblktapctl0-openvstorage           2.0.90-2ubuntu4                     amd64        Xen API blktapctl shared library (shared library)
ii  libvhd0-openvstorage                 2.0.90-2ubuntu4                     amd64        VHD file format access library
ii  libvhdio-2.0.90-openvstorage         2.0.90-2ubuntu4                     amd64        Xen API blktap shared library (shared library)
ii  openvstorage                         2.7.8-rev.4429.a90f470-1            amd64        openvStorage
ii  openvstorage-backend                 1.7.8-rev.873.16660bb-1             amd64        openvStorage Backend plugin
ii  openvstorage-backend-core            1.7.8-rev.873.16660bb-1             amd64        openvStorage Backend plugin core
ii  openvstorage-backend-webapps         1.7.8-rev.873.16660bb-1             amd64        openvStorage Backend plugin Web Applications
ii  openvstorage-core                    2.7.8-rev.4429.a90f470-1            amd64        openvStorage core
ii  openvstorage-hc                      1.7.8-rev.873.16660bb-1             amd64        openvStorage Backend plugin HyperConverged
ii  openvstorage-health-check            3.1.2-rev.346.13d0922-1             amd64        Open vStorage HealthCheck
ii  openvstorage-sdm                     1.6.8-rev.493.d1d73e1-1             amd64        Open vStorage Backend ASD Manager
ii  openvstorage-webapps                 2.7.8-rev.4429.a90f470-1            amd64        openvStorage Web Applications
```

### Steps:
* Stop a volumedriver
* Removed it via the GUI

### Tested on:
* 1 node volumedriver cluster
* 3 node volumedriver cluster
